packages/api-client-core/src/GadgetConnection.ts

```diff
@@ -77,6 +78,32 @@ export enum AuthenticationMode {
 
 const objectForGlobals = typeof globalThis != "undefined" ? globalThis : typeof window != "undefined" ? window : undefined;
 
+const liveQueryExchange: Exchange = ({ forward }) => {
+  const executed = new Set<number>();
```

Comment:
`WeakSet` maybe? Otherwise this will leak if we have any bugs that make things never get torn down